packages/core/src/config/minimal.ts

```diff
@@ -55,5 +56,6 @@ export default {
     'no-unused-components': 'warn',
     'no-undefined-server-variable': 'warn',
     'no-server-variables-empty-enum': 'error',
+    'operation-4xx-problem-details-rfc7807': 'off',
```

Comment:
seems like we don't need to add rules with 'off' here.
Looks like it doesn't work correctly and enables those rules in fact. Check the test snapshots updated above. They should not.